seatunnel-dist/pom.xml

```diff
@@ -636,12 +636,6 @@
                 </dependency>
 
                 <!-- hadoop jar -->
-                <dependency>
-                    <groupId>org.apache.hadoop</groupId>
-                    <artifactId>hadoop-aliyun</artifactId>
-                    <version>${hadoop-aliyun.version}</version>
-                    <scope>provided</scope>
-                </dependency>
```

Comment:
why delete it

Reply:
Because it's repetitive. In the line 665 we already have `hadoop-aliyun`.